.ci/magic-modules/downstream_changelog_metadata.py

```diff
@@ -96,12 +95,9 @@ def set_changelog_info(gh_pull, release_note, labels_to_add):
     print "Skipping, no downstreams repos given to downstream changelog info for"
     sys.exit(0)
 
-  gh = github.Github(os.environ.get('GITHUB_TOKEN'))
   assert len(sys.argv) == 2, "expected id filename as argument"
   with open(sys.argv[1]) as f:
     pr_num = int(f.read())
-    try:
```

Comment:
Why aren't we logging the error anymore?

Reply:
this was a redundant log when i was trying to figure out what was happening (since we are just throwing the error right after, it'll get output anyway) also it was broken